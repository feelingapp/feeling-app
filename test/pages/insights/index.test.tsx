import { shallow } from "enzyme"
import Insights from "../../../src/pages/insights"

describe("Insights: ", () => {
  it("matches snapshot", () => {
    const wrapper = shallow(<Insights />)
    expect(wrapper).toMatchSnapshot()
  })
})
