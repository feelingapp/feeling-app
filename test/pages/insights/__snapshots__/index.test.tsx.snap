// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Insights:  matches snapshot 1`] = `
<Fragment>
  <Head>
    <title>
      Insights / Feeling
    </title>
  </Head>
  <TopBar
    title="Insights"
  />
</Fragment>
`;
